javascript/main.js

```diff
@@ -1,51 +1,60 @@
-var restify = require('restify');
+var restify = require("restify");
 
 var server = restify.createServer();
 server.use(restify.plugins.queryParser());
 
 var DATA_STORE = {};
 
-/*
-*  Serve up a welcome page at the root url
-*/
-server.get('/', function(req, res, next) {
-  res.send('Hello, world. Looks like everything is working. You got this!');
+/**
+ *  Serve up a welcome page at the root url
+ */
+server.get("/", function(req, res, next) {
+  res.send("Hello, world. Looks like everything is working. You got this!", {
+    "content-type": "text/plain"
```

Comment:
This is more of a controversial change. It felt valuable to explicitly use `text/plain` vs. default. I think this makes the key listing more readable (newline vs. comma delimited).